pkg/frontend/v2/frontend.go

```diff
@@ -61,6 +62,7 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet, logger log.Logger) {
 	f.IntVar(&cfg.WorkerConcurrency, "query-frontend.scheduler-worker-concurrency", 5, "Number of concurrent workers forwarding queries to single query-scheduler.")
 
 	cfg.InfNames = netutil.PrivateNetworkInterfacesWithFallback([]string{"eth0", "en0"}, logger)
+	f.BoolVar(&cfg.EnableIPv6, "instance-enable-ipv6", false, "Enable using a IPv6 instance address. (default false)")
```

Comment:
```suggestion
	f.BoolVar(&cfg.EnableIPv6, "query-frontend.instance-enable-ipv6", false, "Enable using a IPv6 instance address. (default false)")
```

Reply:
And please be aware that the `make reference-help` and `make doc` needs to be re-run

Reply:
thanks, updated doc